WakeDevice/TargetEndpoint.cs

```diff
@@ -101,6 +112,12 @@ public TargetEndpoint(string mac, string ip, string? mask, List<uint>? ports)
 
             AddressPorts = ports;
         }
+        
+        /// <summary>
+        /// Wakes target device performing Wake On LAN
```

Comment:
device performing?
"Wakes device by performing" was probably what it was supposed to mean, optionally could be a little more specific (eg. by sending magic packet to each port) but up to dev

Reply:
Fixed by [commit 0fce650a29b8489cd7f4195e49f6f84281429df4 ](https://github.com/Emilkens/WakeDevice/pull/2/commits/0fce650a29b8489cd7f4195e49f6f84281429df4)